rclpy/rclpy/timer.py

```diff
@@ -12,20 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from rclpy.clock import Clock
-from rclpy.clock import ClockType
 from rclpy.handle import Handle
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 from rclpy.utilities import get_default_context
 
 
-# TODO(mikaelarguedas) create a Timer or ROSTimer once we can specify custom time sources
-class WallTimer:
```

Comment:
Doesn't this change break any code which uses `WallTimer`? If yes, I would rather preserve the `WallTimer` class with its current behavior and introduce the `Timer` class beside it.

Reply:
Outside of `rclpy` I'd expect it to only break type annotations. I wouldn't expect `WallTimer` to be used directly since it needs to be part of a `Node` to be served by an executor. It seems likely that all instances of `WallTimer` outside of `rclpy` are being created by `node.create_timer()`.